src/fennel/binary.fnl

```diff
@@ -304,6 +322,11 @@ To include C libraries that contain Lua modules, add --native-module path/to.so,
 and to include C libraries without modules, use --native-library path/to.so.
 These options are unstable, barely tested, and even more likely to break.
 
+If you need to change the require name that a given native module is referenced
+as, you can use the --rename-native-module ORIGINAL NEW. ORIGINAL should be the
+suffix of the luaopen_* symbol in the native module. NEW should be the string
+you wish to pass to require to require the given native module.
```

Comment:
I wonder if an example would help here? I know you ran into the need for this in some commonly-used library so maybe clarifying *why* you might need this would be helpful?